index.md

```diff
@@ -32,76 +53,43 @@ Event
 The PyMC team has recently started hosting office hours regularly.
 Subscribe on Discourse to be notified of the next event!
 :::
-:::{grid-item-card} Welcome to the PyMC blog
-:link: welcome_post
-:link-type: ref
-:class-header: bg-pymc-two
 
-Blog Post
-^^^
-Check out the new PyMC blog, learn what to expect from it and how to subscribe
-:::
 :::{grid-item-card} Probabilistic Programming in PyMC
 :link: https://austinrochford.com/posts/intro-prob-prog-pymc.html
-:class-header: bg-pymc-three
-
+:class-header: bg-pymc-two
```

Comment:
```suggestion
:class-header: bg-pymc-two

```

has no effect but I'd keep an empty line to separate content from directive arguments